hooks/listings/useListings.jsx

```diff
@@ -1,7 +1,7 @@
 import { useQuery } from 'react-query';
 
 async function fetchListingsRequest() {
-	const response = await fetch('/api/listings');
+	const response = await fetch('/api/listings?');
```

Comment:
Oh this isn't what I meant with the question mark. I meant where the code was like
`listings.map` to make it into `listings?.map`. The question mark represents optional chaining, and it acts as a null check, basically if the thing before the question mark is null it won't try to run `map` on it and will just return null.

Reply:
No problem, I'll correct that then.